### ISSUE TITLE
Parameters

### ISSUE THREAD

**Issue by moafzalmulla** [oliger/react-week-calendar#2]
Will be cool if you can show us the best way to pass title, location parameter etc through as theres not much documentation in readme.

**Comment by oliger** [comment]
Yeah, it was not meant to be a long-lived project. Not sure to get what you mean by "best way to pass title, location parameter etc".

**Comment by himanshuc3** [comment]
@tweetzal  probably means making a demo/api to showing how to pass props to show events for faster use of this project.
